Sources/LocalNetworkConnectivity/VLCServerListViewController.m

```diff
@@ -401,11 +401,6 @@ - (void)themeDidChange
     _localNetworkTableView.separatorColor = PresentationTheme.current.colors.background;
     _refreshControl.backgroundColor = PresentationTheme.current.colors.background;
     self.navigationController.view.backgroundColor = PresentationTheme.current.colors.background;
-    if (@available(iOS 13.0, *)) {
```

Comment:
now navigationBar changes color without this code (cause of extension) - and without lag!